File: HorseRacing.Web/src/interfaces/api/contracts/model/auth/responses/authentication/authentication-response-dto.ts
import type { BaseDto } from "~/interfaces/api/contracts/base/dto/base-dto";

export interface AuthenticationResponseDto extends BaseDto {
    /** Код пользователя */
    Id: string;
    /** Логин */
    Username: string;
    /** Токен доступа */
    Token: string;
    /** Имя */
    FirstName: string;
    /** Фамилия */
    LastName: string;
    /** Email */
    Email: string;
}

File: HorseRacing.Contracts/Models/Authentication/Responses/Authentication/AuthenticationResponseDto.cs
using HorseRacing.Contracts.Base.Dto;

namespace HorseRacing.Contracts.Models.Authentication.Responses.Authentication
{
    public class AuthenticationResponseDto : BaseDto
    {
        public Guid Id { get; set; }
        /// <summary>
        /// Логин
        /// </summary>
        public string UserName { get; set; } = string.Empty;
        /// <summary>
        /// Имя
        /// </summary>
        public string FirstName { get; set; } = string.Empty;
        /// <summary>
        /// Фамилия
        /// </summary>
        public string LastName { get; set; } = string.Empty;
        /// <summary>
        /// Почтовый ящик
        /// </summary>
        public string Email { get; set; } = string.Empty;
        /// <summary>
        /// Токен авторизации
        /// </summary>
        public string Token { get; set; } = string.Empty;
    }
}
